client/my-sites/site-settings/settings-security/main.jsx

```diff
@@ -64,7 +65,7 @@ export const SiteSettingsSecurity = ( {
 			<SiteSettingsNavigation site={ site } section="security" />
 			{ showCredentials && <JetpackCredentials /> }
 			{ showJetpackBanner && <JetpackCredentialsBanner siteSlug={ site.slug } /> }
-			<JetpackMonitor />
+			{ ! isEnabled( 'layout/dotcom-nav-redesign' ) && <JetpackMonitor /> }
```

Comment:
```suggestion
			<JetpackMonitor />
```

Sorry, this is my fault, the issue text is saying "move" when it should have said something like "copy".

Non classic view users need to keep access to this even after the redesign is available. Classic view users won't see any links to this settings page from anywhere so we can leave it in both places.